db/data/20240129115821_set_up_sms_reminders_india_feb_apr_2024.rb

```diff
@@ -0,0 +1,62 @@
f 29c12064-4569-4eb1-81c6-e997859bb001 0236a21b-96a0-4d49-86bd-baa87e1620a4 15a1d699-943b-48f8-bb19-166b69bba953 6077516c-d1e3-4cc7-abb5-62fa8121992b 094fb16b-a012-4931-b68e-ac4b8aa7a922 ead96917-f28e-492c-a4f4-b1142b9b9fe5 b758a4c0-ba2b-4bfc-ac62-71fa68c05f59 49ba4066-fc61-475f-bd27-a89b2d2d2f2f 5c914033-d89a-4afa-93da-cc223389e6b8 c1decd73-6013-4a60-b2fe-291c578e97ba 01ee68c9-8a60-46ee-b874-fee5a9e79182 f14383fb-b890-4c6b-bc05-30dea349fc6a eff88eec-abb4-45ba-b667-858afe960b8c 889b9bf5-ba9a-4283-8ef4-36b9ad33c1ad 2028763d-7445-4dba-971e-0e8f7d61441e 8ec44e95-f939-4fb7-9275-cfb96c4e5816 8fd1e480-d5ef-407a-a1c3-b819070c48af e45520c7-2837-4349-9212-bc50cc0ab739 11baa6b7-1351-4710-ab94-01691d12d06c c2562a44-cca1-4f6a-bd87-067ea40b73f2 d59be528-0f13-4f2b-b90e-bf912532da56 0fb8795a-24de-4d8a-b9dd-e473e216d08e 76134ec8-2f5d-4ace-87ea-2b4715cb72c6 b1275ed5-5c3f-4a69-af0c-e9b8c796936b 2cf51c91-27ba-4404-acdf-479d97b3ae3a a4946e66-d703-40d3-8634-656bc7fbc1af 452e733a-fc4d-4847-8594-e68acdc16aa5 86cf25b1-bbf0-46b7-9ac0-2d1699b52615 f299da5b-c26a-4517-8729-bcbbe67728a8 7961fc0a-afe0-43e3-b847-3d643a1cfef5 6983d32f-d4f2-45dc-a604-2ca181e7284e 302f8eb2-224f-4b9c-9b27-21b23b0ac754 358d6aaf-779b-4fa7-a88c-2bda6d7fcdc4 734fb75f-25f5-4608-93b8-e8ab4091a1a3 30cb862b-1002-4562-a9b9-3910ee85acb0 6f09afd5-d878-448b-8d2a-ac0799f40900 230b8bf1-54f9-49ac-98e1-7843dab21819 1e5f5ee9-251e-4e68-a9da-b02fbbef0902 b7ac47c4-db73-4a79-9753-ff2c70a77449 b10be628-29d4-4ccb-a711-38e590ecf03c 91f7817b-f73c-47cf-a8fb-c38280b0be65 e5fcdedc-bb28-4389-be94-286eca535724 3491ff9f-7931-4dff-bea6-22250db6a20d e9566b33-252e-453f-86ad-02d53626041c 1667cd1e-cdef-4387-b418-f9516c34c9f6 e51abd8b-3be3-43db-a86f-0197954263f3 0fff0a31-7450-4e31-8e9f-b1ca858d661d c2b81545-0138-4014-8694-82e570f4ee66 abfb46c0-b8dd-40fe-aa7a-0ed693aad928 cf455f76-0bc0-4f59-b2be-c628de2b85c0 39731fe9-f13a-4370-a148-0968e73ccd16 42ca44b1-2fe8-49d3-af66-33bc2c36763f 8ba7d5df-5e36-4db9-a02c-ad410a6e6e31 90e846bb-d2f7-489f-b758-c5b5fdae3f70 f6395338-4ecb-468c-9a72-d1b5b8e1169e 12077f18-82ff-4d24-abb6-f968f4ae31d2 7b3e5865-dfdd-42c6-ac79-bf1570669591 70ecb380-93b1-4dc0-9aa3-bbebf3df30be 5d652b3f-29db-4e80-b82e-8d690aa16d9f e56e32a6-822a-44c8-aba5-2d066ef9c84b c953f71f-fca9-4456-ac16-a363cafef035 b3c2e413-d9df-49a6-afa0-2bbc4610e884 91be7a95-4dec-4f0d-b2e9-5ddb87f05a6a bf97c3a0-fa91-4b66-857a-374083dcc271 dc0d57ce-239c-4ea0-b3a2-c5aadf683cb1 86bbbcd2-7142-4378-8078-8d101be8932c 0d7e42ac-b4ed-4c24-bfc9-5211ded7a9f0 795048aa-4d6c-49b4-abef-4e73c1a2aecf 9c1f3013-1347-4adb-b027-cfd642d2e20a 5309ff56-7c53-4e7d-9939-d25e2cd2018d dadd4cb6-dc42-4f12-8c3c-c4fa1c8fd46d 7435afbf-28bf-432f-9624-fda905b837c8 4da7d239-b439-4003-9a3f-a242dd4db856 ef6fbb8d-e4b5-4844-a7e3-8627e9328df5 f1dc8d43-4da9-4378-a1f4-a52dbd5294d3 43e60477-65bf-4407-a92d-81a6c727c0dd b6c07887-b600-4840-89c9-90d65386c9d8 ff16e816-893c-4908-890b-bc64bce22c10 179e7cef-c812-4ce0-af6e-766c8c2e15f0 683472ef-f787-4da9-b2d0-0fa34003688e b36a729c-0fea-4406-a63b-edaf26e8a662]}
```

Comment:
If the intent is to include a few districts, why are we excluding a long list of blocks instead? 
This long list of block ids in the code, is a code smell. I think this should be refactored. 

Also you should be very careful when committing ids to the codebase.

Reply:
Using slugs instead of ids or names is better since they are human read able and don't have issues related names either.

Reply:
There's a card for this refactor [here](https://app.shortcut.com/simpledotorg/story/11699/refactor-regions-filter-in-notification-experiments) but it's a bit involved. Would be good to pick it up soon.

This file is copying over values from the first migration that added these values that we know works, in case we want to set this up until the refactor is done.

Reply:
> If the intent is to include a few districts, why are we excluding a long list of blocks instead?
This is how filters are coded right now. Think this requirement with districts wasn't anticipated when it was written.